taca/analysis/analysis.py

```diff
@@ -305,11 +306,15 @@ def _process(run):
                 logger.info(("BCL conversion and demultiplexing process in "
                              "progress for run {}, skipping it"
                              .format(run.id)))
+                ud.check_undetermined_status(run.run_dir, status=run.status, und_tresh=CONFIG['undetermined']['lane_treshold'],
```

Comment:
as this is part of the analysis command, can you place this configs under the `analysis` section in the config file?

```yaml
analysis:
    undetermined:
        lane_threashold: blabla
        q30_threshold: bloblo
```